<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Flights</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>311</width>
        <height>129</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Add Flights:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Flight ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="flightIdEdit">
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string># Seats:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="flightSeatsEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="addFlightButton">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="addFlightStatus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>311</width>
        <height>129</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Remove Flights:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Flight ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rflightIdEdit">
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="removeFlightButton">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="removeFlightStatus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>361</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Query Flight:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Flight ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="queryFlightIdEdit">
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cbOccupied">
         <property name="text">
          <string>Show Occupied Seats Only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSorted">
         <property name="text">
          <string>Sort by Occupant Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="queryFlightButton">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="queryFlightOutput">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Customers</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>236</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Add Customer Seat Reservation</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Seat #:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Flight ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="addCustomerFlightId"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="addCustomerSeatNum">
           <property name="maximum">
            <number>49</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="addCustomerName">
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="placeholderText">
          <string>Customer Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addCustomerAddress">
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="placeholderText">
          <string>Customer Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Customer Phone Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addCustomerPhoneNum">
         <property name="inputMask">
          <string>(999) 999-9999</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addCustomerSubmit">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="addCustomerStatus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>311</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Find Customer Seat Reservation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="findCustomerName">
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="placeholderText">
          <string>Customer Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Customer Phone Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="findCustomerPhoneNum">
         <property name="inputMask">
          <string>(999) 999-9999</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findCustomerSubmit">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="findCustomerStatus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 3</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
